README.md

```diff
@@ -1,8 +1,13 @@
-# BCPL Blog
+## BCPL Blog
 
 Home of the code for the BCPL Blog.
 
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+- [BCPL Blog](#bcpl-blog)
```

Comment:
Shouldn't the table of contents be at the start of the file?

Reply:
yea i thought so too, but it looked funky, ill do a little more research

Reply:
we agreed over chat, this is ok for now.

Reply:
ready for another look